eng/Versions.props

```diff
@@ -64,7 +64,7 @@
        - Feed URI in the nuget.config
        - Native assets build and sha
     -->
-    <_SkiaSharpVersion>2.88.2</_SkiaSharpVersion>
+    <_SkiaSharpVersion>2.88.6</_SkiaSharpVersion>
```

Comment:
@webwarrior-ws are you explaining this change in commit msgs?

Reply:
Yes, see 709a1a3